src/main/java/seedu/address/model/person/DateTime.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.LocalDateTime;
+
+/**
+ * Represents the date and time a Food was recorded.
+ */
+public class DateTime {
```

Comment:
Not sure if we need to override the `hashCode()` method (if it's used in the `argMultiMap` then perhaps we need to override it)?